posthog/api/person.py

```diff
@@ -208,6 +208,16 @@ def list(self, request: request.Request, *args: Any, **kwargs: Any) -> response.
 
         return Response({"results": serialized_actors, "next": next_url, "previous": previous_url})
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "delete_events",
+                OpenApiTypes.BOOL,
+                description="If true, a task to delete all events associated with this person will be created and queued. The task does not run immediately but is processed on the cron pattern set by CLEAR_CLICKHOUSE_REMOVED_DATA_SCHEDULE_CRON",
```

Comment:
```suggestion
                description="If true, a task to delete all events associated with this person will be created and queued. The task does not run immediately and instead is batched together and run once a week",
```

Reply:
We can be more specific here if you want to since we know it's run 5AM UTC on every Sunday (on Cloud)

Reply:
Awesome

Reply:
Will update

Reply:
If I have time I'll have a look into whether we can get this to be using the variable - but will get this merged in furst